RELEASING.md

```diff
@@ -1,3 +1,69 @@
 ## Releasing
 
-This project follows [OpenSearch project branching, labelling, and releasing](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
\ No newline at end of file
+This project follows [OpenSearch project branching, labelling, and releasing](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
+
+## Runbook
+
+### Overview
+
+The OpenSearch project releases as versioned distributions of OpenSearch, OpenSearch Dashboards, and the OpenSearch plugins. This runbook describes in detail how to perform an end-to-end major/minor/patch version release for OpenSearch Dashboards project as the release manager (RM) of this release version. RM is responsible for updating the release status broadly on the release tracking issue on github.
+
+#### Important Dates
+
+- Next minor release: [Version 2.4.0 (November 10, 2022)](https://github.com/orgs/opensearch-project/projects/1#column-19044869)
+- Code freeze date for next minor release: [Version 2.4.0 (November 03, 2022)](https://github.com/opensearch-project/openSearch-build/issues/2649#issue-1378667303)
+
+### Release Phase 1 - Preparation
+
+To track the major / minor / patch release, a github issue will be created. The issue will be created on the OpenSearch Dashboards repository for major / minor releases, with the overall issue tracked on the OpenSearch build repository. Only an overall issue will be created on the OpenSearch build repository for patch release.
+
+(Example github issues of [major](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1548) / [minor](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2230) / [patch](https://github.com/opensearch-project/opensearch-build/issues/2650) release)
+
+The release manager (RM) will be assigned to the OpenSearch Dashboards release issue and will be responsible for reviewing all tasks in this issue for preparation. It's also a good idea to compare the current release issue to the issue of previous release version at this point to ensure that all new processes have been captured.
+
+RM will review the [public roadmap](https://github.com/orgs/opensearch-project/projects/1) and confirm the release scope with other OpenSearch Dashboards [maintainers](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/MAINTAINERS.md) as well as feature owners. All features not ready for current release will be relabeled. For example, if current release version is v2.3.0, all features not ready will be labeled as v2.4.0. (or even later by discussing with the feature owner). After then, RM will need to check all PRs for the current release version are merged into main branch and backported PRs are merged with all CI passed.
```

Comment:
`v2.4.0. (or`

i would drop the 3rd `.`

Reply:
This is good info about labels, but we could emphasize that release labels are intended to clearly communicate which features and fixes are targeted for the upcoming release. So this step is basically an audit to make sure issues that should have the label have it, and those that won't make it don't.